src/mainwindow.cpp

```diff
@@ -132,20 +144,12 @@ void MainWindow::when_ReadingList_toggled(bool state)
     }
 }
 
-void MainWindow::when_libraryPageDisplayed(bool showed)
+void MainWindow::libraryPageDisplayed()
 {
-    auto app = KiwixApp::instance();
-
-    // When library sidebar appeared, or hidden, reading list is always hidden.
-    app->getAction(KiwixApp::ToggleReadingListAction)->setChecked(false);
-
-    if (showed) {
-        mp_ui->sideBar->setCurrentWidget(mp_ui->contentmanagerside);
-        mp_ui->sideBar->show();
-    }
-    else {
-        mp_ui->sideBar->hide();
-    }
+    QAction *readingList = KiwixApp::instance()->getAction(KiwixApp::ToggleReadingListAction);
+    readingList->setChecked(false); // Reading list is always disabled in library view
```

Comment:
I just found another small inconsistency - though the reading list keeps being displayed beside a settings tab it cannot be opened or closed while on a settings tab.

Reply:
> I don't get why the reading list is incompatible with the library view. In a newly started `kiwix-desktop` app, why can't I see the list of bookmarks without having to open a content ~(or settings)~ tab?

On top of my [comment in the issue](https://github.com/kiwix/kiwix-desktop/issues/1078#issuecomment-2059321426), after implementing the single click behavior in commit [445284a](https://github.com/kiwix/kiwix-desktop/pull/1079/commits/445284a264074caf3d3ebf64180fd76fe45d554d), this would not act correctly with the library view (or settings).

> I just found another small inconsistency - though the reading list keeps being displayed beside a settings tab it cannot be opened or closed while on a settings tab.

Can confirm. For me, the reading list should be hidden when viewing the settings as well, but would like feedback on it.

Reply:
> On top of my [comment in the issue](https://github.com/kiwix/kiwix-desktop/issues/1078#issuecomment-2059321426), after implementing the single click behavior (open in same tab) in commit [445284a](https://github.com/kiwix/kiwix-desktop/pull/1079/commits/445284a264074caf3d3ebf64180fd76fe45d554d), this would not act correctly with the library view (or settings).

I think that it will work. `TabBar::openUrl()` opens a new tab when called while the current tab is not a content tab. Thus on a library or settings tab a single-click will open a new tab, which is logical.

Reply:
In that case, that is perfect if we decide that is what we want to do.